upgrading/automated_upgrades.adoc

```diff
@@ -639,255 +410,169 @@ openshift_node_upgrade_post_hook=/usr/share/custom/post_node.yml
 ----
 
 [[upgrade-hooks-available-hooks]]
-==== Available Upgrade Hooks
+==== Available upgrade hooks
 
 [[upgrade-hooks-masters]]
 .Master Upgrade Hooks
 [cols="1,1",options="header"]
 |===
-|Hook Name |Description
+|Hook name |Description
 
 |`openshift_master_upgrade_pre_hook`
 a|- Runs _before_ each master is upgraded.
 - This hook runs against _each master_ in serial.
-- If a task must run against a different host, said task must use
+- If a task must run against a different host, the task must use
 link:http://docs.ansible.com/ansible/playbooks_delegation.html#delegation[`delegate_to` or `local_action`].
 
 |`openshift_master_upgrade_hook`
-a|- Runs _after_ each master is upgraded, but _before_ its service or system restart.
+a|- Runs _after_ each master is upgraded but _before_ its service or system restart.
 - This hook runs against _each master_ in serial.
-- If a task must run against a different host, said task must use
+- If a task must run against a different host, the task must use
 link:http://docs.ansible.com/ansible/playbooks_delegation.html#delegation[`delegate_to` or `local_action`].
 
 |`openshift_master_upgrade_post_hook`
-a|- Runs _after_ each master is upgraded and has had its service or system restart.
+a|- Runs _after_ each master is upgraded and its service or system restarts.
 - This hook runs against _each master_ in serial.
-- If a task must run against a different host, said task must use
+- If a task must run against a different host, the task must use
 link:http://docs.ansible.com/ansible/playbooks_delegation.html#delegation[`delegate_to`
 or `local_action`].
 |===
 
 [[upgrade-hooks-nodes]]
-.Node Upgrade Hooks
+.Node upgrade hooks
```

Comment:
One known issue about the node upgrade hooks - [BZ#1624799](https://bugzilla.redhat.com/show_bug.cgi?id=1624799) , it's worth mentioning here IMO.

Reply:
I added a limitation for this one.